cloud_storage_handler/clients/minio.py

```diff
@@ -0,0 +1,22 @@
+"""Minio client module.
+
+This module provides functionality to interact with the Minio cloud storage
+service.
+"""
+
+from cloud_storage_handler.clients.minio_client import MinioClient
+
+
+def get_minio_client():
```

Comment:
I think this is a misnomer - it should rather be something like `init_minio()` or `init_bucket()`, no? I think if you want to _get_ a client instance, you'd just call `cloud_storage_handler.clients.minio_client.MinioClient().get_client()` (as you actually do in this function).